ui/src/containers/SituationList.vue

```diff
@@ -0,0 +1,68 @@
+<script setup lang="ts">
+import { useSituationsStore } from '@/store/useSituationsStore'
+import SituationCard from '@/components/SituationCard.vue'
+import SituationDetail from '@/components/SituationDetail.vue'
+import { reactive } from 'vue'
+const situationStore = useSituationsStore()
+situationStore.getSituations()
+const state = reactive({
+	selectedSituationIndex: 0
+})
+const situationSelected = (id: string) => {
+	state.selectedSituationIndex = situationStore.$state.situations.findIndex(
```

Comment:
I haven't used Pinia as much as Vuex, but is there a difference in using `.$state.situations` instead of just `.situations` ?

Reply:
for me it didn't work without $state, it was not updating on changes... but I am not sure, of course. I would check it again